src/models/tiles/geometry/geometry-content.ts

```diff
@@ -1251,6 +1269,25 @@ export const GeometryContentModel = GeometryBaseContentModel
       return elems ? elems as JXG.GeometryElement[] : undefined;
     }
 
+    function setSelectedColor (color: number, board: JXG.Board) {
```

Comment:
The name of this function suggests that it is just a setter method for `selectedColor`, but it is also changing the color of objects. I would suggest renaming it, and/or adding a jsdoc comment that says what it does.

Also not a big deal, but just about all of the model methods that take a `board` argument put the `board` argument first.